src/pages/guides/index.md

```diff
@@ -17,9 +17,15 @@ description: Get started building your Photoshop plugin with the UXP Plugin API.
 
 ## Hello World
 
-Building your first UXP plugin requires two pieces of software, both of which can be installed via [Adobe Creative Cloud Desktop](https://www.adobe.com/creativecloud/desktop-app.html) - the latest Photoshop, and the [UXP Developer Tool](./devtool/installation/).
+Building your first UXP plugin requires two pieces of software, both of which can be installed via [Adobe Creative Cloud Desktop](https://www.adobe.com/creativecloud/desktop-app.html) - the latest version of Photoshop, and the [UXP Developer Tool](./devtool/installation/).
 
-While you download and install both, we suggest reading up on the following:
+To get started, we suggest following these tutorials to get your hands dirty and get started with building UXP plugins for Photoshop:
+
+1. [Creating your first plugin](./getting-started/creating-your-first-plugin/index.md)
+2. [Editing the document](./getting-started/editing-the-document/index.md)
```

Comment:
Since the prerequisite of #2 and #3 is #1 anyway, should we take the user **directly** to 'Creating your first plugin' to begin with?

Reply:
Interesting: I thought about doing this myself but then decided against it.

My rationale was that this way, I make it clear to readers that this is a three pages long "journey" (setting the right expectations) while making it clear (with the title of the first bit) "this is where you need to start".

What do you think?